tutorial/createcredential.js

```diff
@@ -4,18 +4,19 @@ var uport = require('../lib/index.js');
 var jsontokens = require('jsontokens')
 
 var app = express();
-var signer = uport.SimpleSigner('955d253bca62bf380e34a36ed3802895e4adc297fa861e22bb7ee3e66714592b')
+var signer = uport.SimpleSigner('28cefd149967e661b38495d2b5ab3964ffa0055912512d7125896646102c025b')
 
 
 var credentials = new uport.Credentials({
   appName: 'Credential Tutorial',
-  address: '0x15bcc15904f2317365cdb6beaf473b2315405763',
-  signer: signer
+  address: '2od4Re9CL92phRUoAhv1LFcFkx2B9UAin92',
+  signer: signer,
+  networks: {'0x4': {'registry' : '0x2cc31912b2b0f3075a87b3640923d45a26cef3ee', 'rpcUrl' : 'https://rinkeby.infura.io'}}
```

Comment:
@christianlundkvist If we leave it, add a comment explaining it. Otherwise it's confusing.